pkg/generate/app/environmentfile.go

```diff
@@ -0,0 +1,38 @@
+package app
+
+import (
+	"bufio"
+	"os"
+)
+
+// IsPossibleEnvironmentFile returns true if the argument is a file. I am not
+// entirely sure, why we are using this function, as this is just a wrapper,
+// we are not doing any additional validations here. This is something that I
+// am borrowing from templates file. Need feedback on this.
+func IsPossibleEnvironmentFile(value string) bool {
+	var fileexists bool = true
+	_, err := os.Stat(value)
+	if err != nil {
+		fileexists = false
+	}
+	return isFile(value) && fileexists
+}
+
+// getEnvironmentVars reads a whole file into memory and returns string
+// array. This could be a problem if we are reading a very large file. I am
+// not putting any validation for file, if it contains a key=value pair as
+// that should be handled in environment variables that we are passing.
+func GetEnvironmentVars(path string) []string {
```

Comment:
I was initially thinking of same thing but when I started looking into code deeper, we already have a validation for key-value pair for environment variables. So, I thought of leaving as is. Even with this, the user will be getting a error message like:

oc new-app openshift/postgresql-92-centos7 --env-file=envfile 
error: environment variables must be of the form key=value: oc new-app openshift/postgresql-92-centos7 --env-file=envfile 
error: environment variables must be of the form key=value: POSTGRESQL_USER\user
error: environment variables must be of the form key=value: POSTGRESQL_PASSWORD\sdasdadapassword


 and the contents of env file looks something like this:

POSTGRESQL_USER\user
POSTGRESQL_DATABASE=db
POSTGRESQL_PASSWORD\sdasdadapassword.

But, if you think the other way around, I can have some validation here as well.